Change spectra layout

borondics opened the pull request:
The saving functionality was changing the graph layout before the save and changing it back afterward. I took this out and made the top and right axes permanent with no values displayed next to the ticks. I also added some margins (we could make this dynamic).

markotoplak commented:
Thanks! I removed your accidental commits. I also removed the line that set the margins, because it could be problematic with longer axis values.

borondics commented:
Thanks for the merge. We will need some kind of margins though because the
last number gets cut off in the bottom right corner. There is probably a
better way of doing it than what I tried to hard-code.

On Fri, Oct 5, 2018 at 5:35 PM Marko Toplak <notifications@github.com>
wrote:

> Thanks! I removed your accidental commits. I also removed the line that
> set the margins, because it could be problematic with longer axis values.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Quasars/orange-spectroscopy/pull/221#issuecomment-427407282>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGA0GjwDOsX_67w4jTxNdJ0JSJcyvVkkks5uh3xLgaJpZM4XIXub>
> .
>
